### PR TITLE
Plural Learner and Generics Curriculum

### PR DESCRIPTION
**Pull request by denizbeser**

### CODE REVIEW

File: adam/curriculum/phase1_curriculum.py [hunk 1]
```diff
@@ -234,7 +236,7 @@ def _make_objects_with_colors_is_curriculum() -> Phase1InstanceGroup:
     )
 
 
-def _make_multiple_objects_curriculum() -> Phase1InstanceGroup:
+def _make_plural_objects_curriculum() -> Phase1InstanceGroup:
     """
     We are deferring handling numeric quantifiers until Phase 2,
```
Comment (LichMaster98) [inline adam/curriculum/phase1_curriculum.py:241]: Perhaps we should update this doc string?

File: adam/curriculum/phase1_curriculum.py [hunk 2]
```diff
@@ -270,6 +275,33 @@ def build_object_multiples_situations(
     )
 
 
+def _make_generic_statements_curriculum() -> Phase1InstanceGroup:
+    # Hard-coded examples: we create dynamic instances and replace the linguistic description
```
Comment (LichMaster98) [inline adam/curriculum/phase1_curriculum.py:279]: Can you add a reference to https://github.com/isi-vista/adam/issues/771

**Comment by gabbard** [comment]
@denizbeser : why was this closed?

**Comment by denizbeser** [comment]
@gabbard This PR is obsolete as it was split into two for generics curriculum (already merged) and a [more advanced plural learner](https://github.com/isi-vista/adam/pull/922) PR

**Comment by gabbard** [comment]
@denizbeser : Got it, thanks.

**Pull request closed without merging**